src/Components/MovieCard/MovieCard.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC } from 'react';
+import { Imovie } from '../Body/Body';
+import './style.scss';
+
+type Tprops = {
```

Comment:
It's better to be consistent with propTypes. 
Since props type won't be exported it could be like everywhere
`type Props = {`
`MovieCard: FC<Props> `

Reply:
Thanks, fixed.